contracts/test/accounts/Holder.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.6.11;
+
+interface IBasicFDT {
+    function withdrawFunds() external;
+}
+
+contract Holder {
+
+    function withdrawFunds(address token) external {
```

Comment:
Lacks contract domain. Further, is this account actually needed?

Reply:
It will for the TODO test, but I'll remove for now to avoid confustion